manager/management/commands/export-data.py

```diff
@@ -30,14 +47,38 @@ def add_arguments(self, parser):
             default=None
         )
 
+        parser.add_argument(
+            '--prepare-removal',
+            dest='prepare_removal',
+            type=bool,
+            help='Saves the records that are exported for future removal',
+            default=False
+        )
+
     def handle(self, *args, **options):
-        self.filename = options['output']
+        """
+        The main process of the command
+        """
+        self.filename = options['filename']
+        self.email = options['email']
         self.before = options['before_date']
+        self.prepare_removal = options['prepare_removal']
+
+        if not self.filename and not self.email:
```

Comment:
Should this be `or` instead of `and` here?

Reply:
If it were or, the entire conditional would evaluate to `True` if one of them wasn't set. We're okay with one of them being set, but not the other. We're okay with both of them being set, in which case, it would currently evaluate to `False` using the `and` we have there now. We only want it to throw an error if neither is set, so `self.filename == None` _and_ `self.email == None`.

Reply:
Ah, I see, that makes sense. Thanks for explaining!